src/state/viewer/reducer.ts

```diff
@@ -32,17 +34,31 @@ const actionToConfigMap: TypeToDescriptionMap = {
     [SET_STATUS]: {
         accepts: (action: AnyAction): action is SetViewerStatusAction =>
             action.type === SET_STATUS,
-        perform: (state: ViewerStateBranch, action: SetViewerStatusAction) => ({
+        perform: (state: ViewerStateBranch, action: SetViewerStatusAction) => {
+            if (action.payload.status !== VIEWER_ERROR) {
+                return {
+                    ...state,
+                    status: action.payload.status,
+                    error: "", // Clear out error
```

Comment:
set this to initial state

Reply:
You mean (pseudocode) `return {...initialState}`? I don't think we want to do that because we want to set the status to whatever the payload.status is.

Reply:
no I meant, `error: initialState.error`